### PR TITLE
io/lab: Fix typos

### PR DESCRIPTION
**Pull request by Tibi09**
Fix found typos in io/lab.

### CODE REVIEW

File: content/chapters/io/lab/content/beyond-network-sockets.md [hunk 1]
```diff
@@ -40,5 +40,5 @@ Everything else is the same.
 
 Now fill in the `TODO`s in `receive_unix_socket.c`.
 Use the example at the bottom of [the UNIX socket man page](https://man7.org/linux/man-pages/man7/unix.7.html) if you get stuck at `bind()`.
-When you're done, run compile and it first and then `send_unix_socket`.
+When you're done, compile it first and then run `send_unix_socket`.
```
Comment (teodutu) [inline content/chapters/io/lab/content/beyond-network-sockets.md:43]:
```suggestion
When you're done, compile the code and then run `send_unix_socket`.
```

File: content/chapters/io/lab/content/pipes.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Use your knowledge of pipes to solve the a CTF challenge.
 Navigate to `support/receive-challenges` and look into the files `receive_pipe.c` and `send_fd_4.c`.
 Modify `receive_pipe.c` so that it creates a pipe, then spawns a child process.
 The child will redirect file descriptor 4 to `stdout` and then `execlp()` `send_fd_4`.
-`send_fd_4` wries the flag to  file descriptor 4 (`pipefd[1]`), so the parent process needs to read it from from `pipedefd[0]`.
+`send_fd_4` writes the flag to  file descriptor 4 (`pipefd[1]`), so the parent process needs to read it from from `pipedefd[0]`.
```
Comment (teodutu) [inline content/chapters/io/lab/content/pipes.md:94]:
```suggestion
`send_fd_4` writes the flag to file descriptor 4 (`pipefd[1]`), so the parent process needs to read it from `pipedefd[0]`.
```

**Review by teodutu — CHANGES_REQUESTED** [review]
Thanks for the fixes @Tibi09. I've made some extra corrections. Besides those, add a description to your commit message to pacify `checkpatch.pl` [1]. The `super-linter` failure is a false positive [2]. Let's skip it.

[1] https://github.com/open-education-hub/operating-systems/actions/runs/5026142993/jobs/9021452171?pr=308#step:7:48
[2] https://github.com/open-education-hub/operating-systems/actions/runs/5026142995/jobs/9021452166?pr=308#step:4:415

**Comment by Tibi09** [comment]
Thank you! I've added the corrections and the description. It should be fine now.

**Comment by Tibi09** [comment]
Found a bonus typo :smile:

**Review by teodutu — CHANGES_REQUESTED** [review]
You also need to sign-off your commit [1] like so `git commit --amend --signoff` and then force-push it once again. Make sure you have configured your username and email before signing the commit (`git config --global user.email your@email.com` and `git config --global user.name "Your Name"`).

[1] https://github.com/open-education-hub/operating-systems/actions/runs/5033759696/jobs/9030441388?pr=308

**Review by teodutu — APPROVED** [review]

**Pull request merged**